src/userCLI/src/main/kotlin/org/vaccineimpact/orderlyweb/userCLI/AddUserGroup.kt

```diff
@@ -2,10 +2,13 @@ package org.vaccineimpact.orderlyweb.userCLI
 
 import org.vaccineimpact.orderlyweb.db.OrderlyAuthorizationRepository
 
-fun addUserGroup(options: Map<String, Any>): String
+fun addUserGroups(options: Map<String, Any>): String
 {
-    val groupName = options["<name>"].getStringValue()
+    val groupNames = options["<name>"] as List<*>
 
-    OrderlyAuthorizationRepository().createUserGroup(groupName)
-    return "Saved user group '$groupName' to the database"
+    return groupNames.joinToString("\n") {
```

Comment:
The addUsers code checks if each user already exists, but I think this will throw an Exception if one of the groups already exists, which is probably undesirable behaviour when adding multiple as some will be added and others not. So make this consistent with addUsers?